### PR TITLE
add proposed Code of Conduct

### PR DESCRIPTION
**Pull request by jorydotcom**
This PR adds a proposal for #1 - adopting a code of conduct

### CODE REVIEW

**Comment by dtex** [comment]
Thank you @jorydotcom

**Review by leobalter — APPROVED** [review]

**Review by vlevantovsky — APPROVED** [review]

**Comment by jorydotcom** [comment]
thank you @vlevantovsky @leobalter - we've been moving forward with this as the CoC process for the past couple of meetings, so I think it's safe to merge, but will await @phoddie 's +1 before doing so.

**Comment by leobalter** [comment]
sure thing, @jorydotcom! I'll also wait on EcmaTC53/draft#1 until this one is merged.

**Review by phoddie — APPROVED** [review]

**Pull request merged**
